### DISCUSSION TITLE
硬體需求

### DISCUSSION THREAD

**Discussion by xjlin0** [rnchg/APT#12] [Q&A]
最低運行的硬體需求是什麼?
建議的運行硬體需求是什麼?

**Comment by rnchg** [comment]
Hardware Supported

The models are tested on:

GPU SKU: RTX 4090 (DirectML)
~~GPU SKU: 1 A100 80GB GPU, SKU: Standard_ND96amsr_A100_v4 (CUDA)~~
CPU SKU: Standard F64s v2 (64 vcpus, 128 GiB memory)
Mobile SKU: Samsung Galaxy S21

Minimum Configuration Required:

Windows: DirectX 12-capable GPU and a minimum of 4GB of combined RAM
~~CUDA: NVIDIA GPU >= 7.0~~
